LEAF_Request_Portal/admin/Group.php

```diff
@@ -329,8 +514,33 @@ public function getHistory($filterById)
      *
      * @return array all history ids for all groups
      */
-    public function getAllHistoryIDs()
+    public function getAllHistoryIDs(): array
     {
         return $this->dataActionLogger->getAllHistoryIDs("groupID", \LoggableTypes::PORTAL_GROUP);
     }
+
+    /**
+     * retrieve all the groups
+     *
+     * @return array|false
+     *
+     * Created at: 9/12/2022, 10:22:25 AM (America/New_York)
+     */
+    public function getAllGroups(): array|bool
+    {
+        return $this->db->prepared_query('SELECT * FROM groups WHERE groupID > 1 ORDER BY groupID', array());
```

Comment:
This shouldn't be a select *.  Additionally, the query should be in the body to confirm what's being returned.